build/ansible/roles/init-admin-password-ami/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+- name: Fetch instance metadata
+  uri:
+    url: http://169.254.169.254/latest/meta-data/instance-id
```

Comment:
Wonder if the URL here is meant to be fixed too.

Reply:
No, URL is correct, it's internal URL used in AWS to get metadata.